pkgs/development/python-modules/pyutilib/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, nose
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "PyUtilib";
+  name = "${pname}-${version}";
+  version = "5.6.2";
+
+  propagatedBuildInputs = [ nose six ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "17rynkkq728b7iy65m2rz9gcycipk0r8xiplfb5lvg5l27zm52gv";
+  };
+
+  doCheck = false;
+
+  meta = with lib; {
+    homepage    = "https://github.com/PyUtilib/pyutilib";
+    description = ''
+      The PyUtilib project supports a collection of Python utilities, including
+      a well-developed component architecture and extensions to the PyUnit testing
+      framework.
+    '';
+    license     = licenses.bsdOriginal;
+    maintainers = with maintainers; [ ericsagnes ];
```

Comment:
I think if you don't set platforms hydra won't build it

Reply:
No need to set `platforms` when using `buildPython*`. It is set to the appropriate platforms by default.